### PR TITLE
add documentation about _ERROR columns

### PR DESCRIPTION
**Pull request by DocOtak**

### CODE REVIEW

**Comment by cberys** [comment]
Looks good! 
One small fix
this : Any parameter which numeric values
should be : Any parameter with numeric values
which -> with

File: docs/gen_paramters_doc.py [hunk 1]
```diff
@@ -40,6 +40,18 @@
     Current quality flags are: :ref:`bottle <Bottle Quality Codes>`, :ref:`water <Water Quality Codes>`, :ref:`ctd <CTD Quality Codes>`.
     See the :ref:`Quality Codes` section for more information
 
+Parameter Error
+---------------
+Any parameter which numeric values MAY have an associated error column which takes the form:
```
Comment (cberys) [inline docs/gen_paramters_doc.py]: Looks good! 
One small fix
this : Any parameter which numeric values
should be : Any parameter with numeric values
which -> with

**Comment by DocOtak** [comment]
I think we'll probably not merge this in favor of some explicit mapping (#22) since the taget has moved a bit from "improve exchange" to more of a "document what we have".

**Comment by DocOtak** [comment]
This PR was superseded by #23

**Pull request closed without merging**